pkg/kubelet/config/config_test.go

```diff
@@ -189,8 +189,8 @@ func TestInvalidPodFiltered(t *testing.T) {
 	channel <- podUpdate
 	expectPodUpdate(t, ch, CreatePodUpdate(kubetypes.ADD, TestSource, CreateValidPod("foo", "new")))
 
-	// add an invalid update
```

Comment:
since there is no more invalid update, method name may be changed to `TestSameNamePodFiltered`?

Reply:
The internal function being tested is still filterInvalidPod, so I appreciate that matching name.